Readme.md

```diff
@@ -0,0 +1,194 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./images/projectLogo.png" alt="logo" width="170"  height="auto" />
+  <br/>
+
+  <h3><b>Haiti tec Summit </b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Haiti-tec-summit] <a name="about-project"></a>
+
+
+**[Haiti-tec-summit]** is a app who display information about a big event made for selling tickets for the event  
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Html</li>
+  </ul>
+  <ul>
+    <li>Css</li>
+  </ul>
+  <ul>
+    <li>Javascript</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Display_Event_information]**
+- **[Responsive_mobile_min-width-375px]**
+- **[Responsive_desktop_min-width-768px]**
+- **[Responsive_desktop_width->768px]**
+- **[Online-deployment]**
+- **[Display-menu-link]**
+- **[Hide-and-show-window-view]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://fredo509.github.io/capstone-summit/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly make sure your video **contains a highlight about a piece of the code you find interesting or you are proud of making**.
Note: Once you have recorded a new video, you should make sure it is still keeping the max length of 5 minutes